src/frontend/src/components/MyFeedsPage/DeleteFeedDialogButton.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogActions,
+  DialogContentText,
+  DialogTitle,
+  IconButton,
+} from '@material-ui/core';
+import { Delete } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  button: {
+    padding: '3px 0 3px 0',
+  },
+}));
+
+function DeleteFeedDialogButton({ feed }) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
```

Comment:
Minor nit, but I guess we should be consistent across the whole site with `React.useState` or `React.useEffect` vs doing it in the import `import React, { useState } from 'react'`

Reply:
Good spot, fixed in https://github.com/Seneca-CDOT/telescope/pull/933/commits/3a4bfaec30edef643f85006e0ddd078a10c0ecac